### PR TITLE
Add bootstrap-rating-input@0.4.0 w/ git auto-update

### LINKED ISSUE #10302: [Request] Add bootstrap-rating-input
**Library name:** bootstrap-rating-input
**Git repository url:** https://github.com/javiertoledo/bootstrap-rating-input
**npm package url(optional):** 
**License(s):** MIT 
**Official homepage:** N/A
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10302

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/javiertoledo/bootstrap-rating-input
 * Official website (optional, not the repository): 
 * NPM package url (optional): 
 * License and its reference: [MIT](https://github.com/javiertoledo/bootstrap-rating-input/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 20
   - Count of stars: 305
   - Count of forks: 142
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

 The first line of commit message is NOT less then 50 chars, but clean and easy to understand.

close #10302, cc @javiertoledo

### CODE REVIEW

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t please help review this PR, thanks.

File: ajax/libs/bootstrap-rating-input/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "input"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/javiertoledo/bootstrap-rating-input.git",
+    "fileMap": [
+      {
+        "basePath": "bulid",
+        "files": [
+          "bootstrap-rating-input*.+(js|css|map)"
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-rating-input/package.json]: Let's use `"**/*"` here.

**Comment by extend1994** [comment]
@pvnr0082t I fetched the `src` folder and modified the rule of basepath, please help me review it, thank you.

File: ajax/libs/bootstrap-rating-input/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      {
+        "basePath": "bulid",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "**/*"
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-rating-input/package.json]: @extend1994 It's a little bit not safe or no sense to use `**/*` under a non-dist folder.

Comment (extend1994) [inline ajax/libs/bootstrap-rating-input/package.json]: @pvnr0082t how about `	bootstrap-rating-input*`?

Comment (PeterDaveHello) [inline ajax/libs/bootstrap-rating-input/package.json]: Both better.

**Comment by extend1994** [comment]
@pvnr0082t @maruilian11 please help me review this pull request, thank you.

**Comment by maruilian11** [comment]
Where is the non-minified file? Review yourself before you ping us.

**Comment by extend1994** [comment]
@pvnr0082t @maruilian11 I thought I have added it, sorry for that. please help me review this pull request, thank you.

File: ajax/libs/bootstrap-rating-input/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "bootstrap",
+    "rating",
+    "input"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/javiertoledo/bootstrap-rating-input.git",
+    "fileMap": [
+      {
+        "basePath": "bulid",
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-rating-input/package.json]: Wrong spelling, didn't you test the config by auto-updater?

Comment (maruilian11) [inline ajax/libs/bootstrap-rating-input/package.json]: Please test the auto-update before you send the commit.

Comment (pvnr0082t) [inline ajax/libs/bootstrap-rating-input/package.json]: Sorry about my careless review.

**Comment by extend1994** [comment]
@PeterDaveHello  @pvnr0082t @maruilian11 Sorry for wrong spell and carelessness, please help me review this pull request, thank you.

**Pull request merged**